src/main/java/org/codeforamerica/formflowstarter/ScreenController.java

```diff
@@ -399,11 +507,18 @@ ModelAndView submit(
     return new ModelAndView(String.format("redirect:/%s/%s/navigation", flow, screen));
   }
 
+  /**
+   * Chooses the next screen template and model to render based on what is next in the flow.
+   *
+   * @param flow        The current flow name, not null
+   * @param screen      The current screen name in the flow, not null
+   * @param httpSession The current httpSession, not null
+   * @return the screen template with model data, returns error page on error
+   */
   @GetMapping("{flow}/{screen}/navigation")
   RedirectView navigation(
       @PathVariable String flow,
       @PathVariable String screen,
-      @RequestParam(required = false, defaultValue = "0") Integer option,
```

Comment:
This deletion is questionable, let me know if you'd like it back in there.

Reply:
It's a relic from the previous implementation of this endpoint that existed in MNBenefits but we don't need it here. Feel free to remove!